libs/akita/src/__tests__/setup.ts

```diff
@@ -109,3 +109,6 @@ export function createWidget(id) {
 }
 
 export const tick = () => new Promise(process.nextTick);
+
+@StoreConfig({ name: 'disabledTrackingConfig', disableTracking: true })
```

Comment:
I prefer not using devtools configuration in the store. It should be in the devtools options.[ ](https://datorama.github.io/akita/docs/enhancers/devtools/#options)

Reply:
Ok, so my idea was to enable using Akita in libraries for complex components. This way I could disable it by default and the application developer wouldn't see them unless he chose to through let's say component's module configuration. I suppose you want me to add storeBlacklist to devtools options? In that case i should do it by providing storeBlacklistPattern since i will be relying on storeName's prefix.

Reply:
Add storeBlacklist and support both strings and cb function.

Reply:
And add yourself to the contributors' list

Reply:
Ok, deal.

Reply:
I'll create a new PR